src/edu/csus/ecs/pc2/core/report/ResultsTSVReport.java

```diff
@@ -0,0 +1,65 @@
+package edu.csus.ecs.pc2.core.report;
+
+import java.io.PrintWriter;
+
+import edu.csus.ecs.pc2.core.log.Log;
+import edu.csus.ecs.pc2.core.model.Filter;
+import edu.csus.ecs.pc2.exports.ccs.ResultsFile;
+
+/**
+ * results.tsv report.
```

Comment:
Same comment here as for `ResultsExportReport`:  It would be nice if the header comment gave a little more explanation of what this class is for. For example, it's not at all clear why there are several different Report classes all having to do with "results reporting".  Some documentation explaining what each one provides (and how they differ) would sure help the rest of us who are faced with going through code to figure it out.

Reply:
comment removed

Reply:
Again, I don't know what to say when someone responds to a suggestion that _more_ documentation would be desirable by _removing what little documentation existed._